nvi/validation_nvi_resource.feature

```diff
@@ -2,70 +2,74 @@ Feature: Validation of an NVI resource
-		When a Curator uses the option to view the list Nominated Resources
-		And the Curator asserts that Resources Validated by own Institution should be listed
-		Then the Curator sees a list of all Resources that are Validated by at least one other Institution, including their own Institution
-		And there is an option to inspect the Resource
-		And there is an option to Validate each Resource on behalf of their Institution
+		Examples:
+		| Status                                      | Own institution | Other institutions         | 
+		| Candidate                                   | Candidate       | No status                  | 
+		| Candidate - Waiting for your institution    | Candidate       | Approved                   |
+		| Being checked                               | Being checked   | No status                  |
+		| Being checked - Waiting for your institution| Being checked   | Approved                   | 
+		| Approved                                    | Approved        | No status                  | 
+		| Approved - Waiting for other institution    | Approved        | Being checked              | 
+		| Rejected                                    | Rejected        | No status                  | 
+		| Rejected - Waiting for other institution    | Rejected        | Candidate or Being checked |
+		| Dispute                                     | Rejected        | Approved                   |
+		| Dispute                                     | Approved        | Rejected                   |
+		| Dispute                                     | Candidate       | Dispute                    |
 
-	@needJiraTag 
-	Scenario: Curator views list of NVI-report Candidates
-		When a Curator uses the option to view the list Candidate Resources
-		Then the Curator sees a list of Resources that fulfill the criteria to be NVI Resources
-		And the Resources have authors that are affiliated with the Curator's Institution
-		And no other Institution has Validated the Resource
+	Scenario: The progress bar display the current NVI-report status
+		When the User wish to see details 
+		Then the User may select "Show reporting status"
 
-	@needJiraTag 
-	Scenario: Curator inspects a Resource from the list of Nominated Resources
-		Given a Curator views the list of Resources Nominated to be part of the NVI-Report
-		When the Curator uses the option to view details about a Resource
-		Then the Curator sees a list with Validation statuses for all affiliated Institutions 
-		And there is an option to Validate the Resource on behalf of their Institution
+	Scenario: Show reporting status
+  		When the User select "Show reporting status"
+  		Then the User see a table displaying status for the current open NVI-periode by default
+  		And the columns show NVI resource statuses
+  		And the rows represent my institution's subunits
+  		And I can select to view any previous year
+  		And I has an export option
```

Comment:
This isn't 100% correct, but my attempt to re-phrase it became extremely verbose. Maybe we can do a separate PR later and split this into a `Feature`? Main points:
- "any previous year" => "any reporting year"?
- "... for the current open NVI-periode by default" => The year filter is "sticky" between pages and will not change when switching between table and search pages.